src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+import Separator from 'components/Breadcrumb/Separator/Separator';
+import { breadcrumbItemStyles } from './BreadcrumbItem.style';
+import useTheme from 'hooks/useTheme';
+import { BreadcrumbItemData } from '../Breadcrumb';
+
+type Props<T> = {
+  childComponent: T;
+  isLastItem: boolean;
+  separatorContent: '*' | '>' | '/';
+  clickHandler?: () => void;
+};
+
+const BreadcrumbItem: React.FC<Props<BreadcrumbItemData | React.ReactNode>> = props => {
```

Comment:
the generic in Props i think is not right? ReactNode and BreadcrumbItemData are not the same or event partially equal

Reply:
The original purpose was to pas a generic type to the breadcrumbItem component in order to make the implementation more flexible to other types of data. As you correctly notice, this is not necessary anymore as childComponent will always be a React.ReactNode type.

Besides the above, I also think that I implemented it wrong... The main idea was something like this:
```
type Props<T extends React.ReactNode> = {
  childComponent: T;
  isLastItem: boolean;
  separatorContent: SeparatorStyle;
};

const BreadcrumbItem = <T extends React.ReactNode>(props: Props<T>) => ...
```